easybuild/easyconfigs/l/libevent/libevent-2.0.22-GCC-5.4.0-2.26.eb

```diff
@@ -0,0 +1,19 @@
+easyblock = 'ConfigureMake'
+
+name = 'libevent'
+version = '2.0.22'
+
+homepage = 'http://libevent.org/'
+description = """The libevent API provides a mechanism to execute a callback function when a specific
+ event occurs on a file descriptor or after a timeout has been reached.
+ Furthermore, libevent also support callbacks due to signals or regular timeouts."""
+
+toolchain = {'name': 'GCC', 'version': '5.4.0-2.26'}
+
+source_urls = [
+    'https://github.com/%(name)s/%(name)s/releases/download/release-%(version)s-stable/',
```

Comment:
tmux-2.2-GCCcore-4.9.3.eb already says source_urls = ['https://github.com/tmux/tmux/releases/download/%(version)s/']: https://github.com/hpcugent/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/t/tmux/tmux-2.2-GCCcore-4.9.3.eb

But I have now changed tmux/tmux to %(name)s/%(name)s. I hope that's what you mean.

So I just made the others say the same thing...